launch/launch/actions/execute_local.py

```diff
@@ -182,6 +183,8 @@ def __init__(
         :param: respawn if 'True', relaunch the process that abnormally died.
             Either a boolean or a Substitution to be resolved at runtime. Defaults to 'False'.
         :param: respawn_delay a delay time to relaunch the died process if respawn is 'True'.
+        :param: respawn_max_retries number of times to respawn the process.
+            Defaults to infinite times.
```

Comment:
```suggestion
            A negative value will respawn an infinite number of times (this is the default behavior).
```

Reply:
Replicated the same suggestion on `execute_process.py`